schema.sql

```diff
@@ -5,4 +5,5 @@ CREATE TABLE animals (
     escape_attempts INT,
     neutered BOOLEAN,
     weight_kg DECIMAL
+    species VARCHAR(255)
 );
```

Comment:
- [ ] I will encourage you to use `ALTER` to achieve this as required, it would look like `ALTER TABLE animals ADD species varchar(255);`